src/hooks/useClickableProps/__stories__/useClickableProps.stories.mdx

```diff
@@ -0,0 +1,275 @@
+import { useCallback, useRef } from "react";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link, Tip } from "../../../storybook";
+import useClickableProps from "../useClickableProps";
+import "./useClickableProps.stories.scss";
+
+<Meta title="Hooks/useClickableProps" />
+
+# useClickableProps
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Return props for adding clickable functionality to the element except for the styles and classNames.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const ref = useRef(null);
+      const onClickCallback = useCallback(() => alert("click!"), []);
+      const clickableProps = useClickableProps(
+        {
+          onClick: onClickCallback,
+          id: "clickable-id",
+          ariaHidden: false,
+          ariaHasPopup: false,
+          ariaExpanded: false
+        },
+        ref
+      );
+      return (
+        <div {...clickableProps} className="monday-storybook-use-clickable-props">
+          I act like a button
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+<Tip title="Alternative">
+  Most of the times you might want to use
+  <Link size={Link.sizes.SMALL} href="/?path=/docs/accessibility-clickable--overview">
+    Clickable component
+  </Link>
+  to apply the styles as well. Use this hook when you want to customize styles by yourself.
+</Tip>
+
+## Arguments
+
+<FunctionArguments>
+  <FunctionArgument
+    name="onClick"
+    type="(event: React.MouseEvent | React.KeyboardEvent) => void"
+    description={<>Click callback.</>}
+  />
+  <FunctionArgument
+    name="onMouseDown"
+    type="(event: React.MouseEvent) => void"
+    description={<>Mouse down callback.</>}
+  />
+  <FunctionArgument name="disabled" type="boolean" description={<>Whether element is disabled or not</>} />
+  <FunctionArgument name="id" type="string" description={<>Id of the element</>} />
+  <FunctionArgument name="dataTestId" type="string" description={<>Id of the element for test purposes</>} />
+  <FunctionArgument
+    name="role"
+    type="string"
+    description={
+      <>
+        Provide semantic meaning to content.
+        <Link size={Link.sizes.SMALL} href={"https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles"}>
+          More details
+        </Link>
+      </>
+    }
+  />
+  <FunctionArgument
+    name="tabIndex"
+    type="number"
+    description={
+      <>
+        Specifies the tab order of an element.
+        <Link
+          size={Link.sizes.SMALL}
+          href={"https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex"}
+        >
+          More details
+        </Link>
+      </>
+    }
+  />
+  <FunctionArgument
+    name="ariaLabel"
+    type="string"
+    description={
+      <>
+        Defines a string value that labels an interactive element.
```

Comment:
Maybe too long? but I think it will be clearer this way... wdyt? 

Aria label is an attribute used in HTML to provide a human-readable label for interactive elements without visible text. This attribute is important for improving web accessibility, specifically for users who rely on screen readers, as it allows them to understand the purpose and function of the element.

Reply:
I think it's too long - we are providing links to read about it and I assume that most of our users are aware about `aria-label` purpose